ebusd/config.yaml

```diff
@@ -14,27 +14,43 @@ arch:
   - i386
 init: false
 startup: "system"
+services:
+ - mqtt:need
 boot: "auto"
+watchdog: tcp://[HOST]:8888
```

Comment:
does it matter what data is presented to the watch dog? I couldn't find any documentation on this.

Reply:
I have pasted two links below.  You just need an active connection, no data is required.

https://www.home-assistant.io/blog/2020/09/16/supervisor-joins-the-party/
https://developers.home-assistant.io/docs/add-ons/configuration

<html>
<body>
<!--StartFragment-->

watchdog | string |   | An URL for monitor an application this add-on. Like http://[HOST]:[PORT:2839]/dashboard, the port needs the internal port, which will be replaced with the effective port. It is also possible to bind the protocol part to a configuration options with: [PROTO:option_name]://[HOST]:[PORT:2839]/dashboard and it's looked up if it is true and it's going to https. For simple TCP port monitoring you can use tcp://[HOST]:[PORT:80]. It work for add-ons on host or internal network.
-- | -- | -- | --


<!--EndFragment-->
</body>
</html>